config/foreman-answers.yaml

```diff
@@ -23,7 +23,6 @@ foreman::cli::tasks: false
 foreman::cli::templates: false
 foreman::cli::webhooks: false
 foreman::compute::ec2: false
-foreman::compute::gce: false
```

Comment:
If you're dropping gce, you'll also need to reflect that in the migration. You could even use the existing answer to determine if the google plugin should be enabled.